### PR TITLE
feat: add args for controller, node, scheduler, lvmd to helm chart

### PR DESCRIPTION
**Pull request by maxime1907**
Provide an `args` option to easily specify the log level for instance of each component

### CODE REVIEW

File: charts/topolvm/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ## [Unreleased]
 
+## [9.2.0] - 2022-10-05
```
[the remark this answers is not in the document]
Comment (maxime1907) [reply charts/topolvm/CHANGELOG.md:10]: okay

File: charts/topolvm/Chart.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ type: application
 home: https://github.com/topolvm/topolvm
 name: topolvm
 description: Topolvm
-version: 9.1.0
+version: 9.2.0
```
[the remark this answers is not in the document]
Comment (maxime1907) [reply charts/topolvm/Chart.yaml:7]: okay

File: charts/topolvm/templates/controller/deployment.yaml [hunk 3]
```diff
@@ -41,7 +41,9 @@ spec:
             - --cert-dir=/certs
             {{- if .Values.controller.nodeFinalize.skipped }}
             - --skip-node-finalize
-            {{- end }}
```
[the remark this answers is not in the document]
Comment (maxime1907) [reply charts/topolvm/templates/controller/deployment.yaml:44]: ah yes mb

**Comment by maxime1907** [comment]
> To update charts/topolvm/README.md, Could you run `make generate-helm-docs`?

done

**Review by toshipp — APPROVED** [review]

File: charts/topolvm/CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,9 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ## [Unreleased]
 
+### Added
+- feat: add `args` for `controller`, `node`, `scheduler`, `lvdm` to helm chart (#576)
```
Comment (daichimukai) [inline charts/topolvm/CHANGELOG.md:11]:
```suggestion
- feat: add `args` for `controller`, `node`, `scheduler`, `lvmd` to helm chart (#576)
```

**Review by daichimukai — CHANGES_REQUESTED** [review]
Please fix typo in PR title as well.

**Comment by maxime1907** [comment]
> Please fix typo in PR title as well.

done! @daichimukai

**Review by toshipp — APPROVED** [review]

**Review by daichimukai — APPROVED** [review]
LGTM

**Pull request merged**